package.json

```diff
@@ -18,7 +19,7 @@
     "knip": "knip --config .github/knip.ts",
     "knip-ci": "knip --no-exit-code --reporter json --config .github/knip.ts",
     "prepare": "husky install",
-    "test": "jest --setupFiles dotenv/config --coverage",
+    "test": "NODE_OPTIONS=\"$NODE_OPTIONS --experimental-vm-modules\" jest --setupFiles dotenv/config --coverage",
```

Comment:
@Ubiquityos what is `--experimental-vm-modules` and is it necessary here?

Reply:
Experimental modules allows for `ESM` usage with `Jest`, required by `Octokit`. https://jestjs.io/docs/ecmascript-modules